scripts/pancancer_classifier.py

```diff
@@ -43,8 +43,6 @@
                             default: False if flag omitted
     --x_matrix          string of which feature matrix to use
                             default: raw
-    --y_matrix          string of which status matrix to use
```

Comment:
so y_matrix is always pancan_mutation_freeze? any reason for it?

Reply:
`pancan_mutation_freeze` is a large sample by gene matrix indicating mutation status for all of TCGA. Previously, the functionality included using the data provided by `xena`. At the time, `xena` was open source TCGA, but the PanCanAtlas was embargoed (updated data, mutation calls, etc.). Now that TCGA is public, we will only provide the options to use this updated data.

In the future, it would be nice to allow updates to this matrix (i.e. a different way to define positive samples). For the sake of this PR being for refactoring, I will wait to introduce this feature in a new PR.